Source/swiftlint/Extensions/Reporter+CommandLine.swift

```diff
@@ -1,6 +1,13 @@
 import SwiftLintFramework
 
 extension Reporter {
+    /**
+        Optionally reports violations as they are found.
+             
+        - Parameters:
+            - violations: An array of `StyleViolation` to report.
+            - realtimeCondition: When enabled, output the report as violations are found.
+    */
```

Comment:
Suggestions:

```swift
/// Reports violations in the `Reporter` type’s format by printing them to standard out.
/// Optionally prints the violations in real time as they are found if the `realtimeCondition`
/// parameter is true. If set to false, the violations will be logged after all files have been
/// processed.
///
/// - parameter violations:        An array of `StyleViolation` to report.
/// - parameter realtimeCondition: If true, will report violations as they are found.
///                                Otherwise, will report violations after all files have been processed.
```